src/views/components/task-view/task-view.js

```diff
@@ -1,38 +1,38 @@
-import React, {Component, Fragment} from 'react';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import { createSelector } from 'reselect';
-import { getAuth } from 'src/auth';
-import {includes} from "lodash/collection";
-import Icon from '../../atoms/icon';
-import { Textbox } from 'react-inputs-validation';
-import TagsInput from 'react-tagsinput';
-import { getCommentList } from 'src/comments';
-import Select from 'react-select';
-import CommentList from '../comment-list';
-import TagsSuggestions from '../tags-suggestions';
-import AddComment from '../add-comment/add-comment';
-import TaskViewHeader from '../task-view-header/task-view-header';
-import i18n from '../../../i18n';
-import {notificationActions} from '../../../notification';
-import { TakeOwnershipModal }  from '../take-ownership-modal';
+import React, { Component, Fragment } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { createSelector } from "reselect";
+import { getAuth } from "src/auth";
+import { includes } from "lodash/collection";
+import Icon from "../../atoms/icon";
+import { Textbox } from "react-inputs-validation";
+import TagsInput from "react-tagsinput";
+import { getCommentList } from "src/comments";
+import Select from "react-select";
+import CommentList from "../comment-list";
+import TagsSuggestions from "../tags-suggestions";
+import AddComment from "../add-comment/add-comment";
+import TaskViewHeader from "../task-view-header/task-view-header";
+import i18n from "../../../i18n";
+import { notificationActions } from "../../../notification";
```

Comment:
@weizenberg In the case you are changing the formatting. Can you do that in two commits? that would make it easier for me to see the actual changes. Right now there are a lot of changes.
Another idea is if you have an idea to do a one time formatting for the whole project?